### PR TITLE
Remove :not pseudo-classes

### PR DESCRIPTION
**Pull request by icidasset**
Hey 👋 

Made a quick adjustment that removes the `:not()` pseudo-classes as well.
~~Also changes the other pseudo-classes-removal part to remove multiple instances.~~

### CODE REVIEW

**Comment by sparksp** [comment]
Hey @icidasset, thanks for making a PR for this. Do you have any example selectors from tailwind that needs these changes? It would be good to add them to [tests.js](https://github.com/monty5811/postcss-elm-tailwind/blob/master/test/tests.js) to make sure we don't break this in the future.

**Comment by icidasset** [comment]
Valid point 😅  I've added some tests.
I don't have any examples from Tailwind itself, but I did have trouble with a plugin I wrote myself.

**Comment by icidasset** [comment]
@sparksp Are these tests sufficient or do I need to write more?

File: helpers.js [hunk 1]
```diff
@@ -110,6 +109,17 @@ function fixClass(cls) {
...
 }
 
+function removePseudoClass(cls) {
+  const result = cls.replace(
+    /(:(active|after|before|checked|disabled|focus|focus-within|hover|visited|nth-child\((even|odd)\)|(first|last)-child))+$/,
+    ""
+  );
+
+  // Remove multiple by doing recursion when the result differs
+  if (result !== cls) return removePseudoClass(result);
```
Comment (sparksp) [inline helpers.js:119]: When I remove this line none of the tests fail - the regex is already grabbing chained selectors with `+` - do you have any other cases where the regex isn't enough?

File: test/tests.js [hunk 2]
```diff
@@ -66,12 +66,24 @@ describe("fixClass", () => {
...
+    });
     it("removes :visited", () => {
       assert.equal(
         h.fixClass(".visited:tw-bg-teal-100:visited"),
         "visited:tw-bg-teal-100"
       );
     });
+    it("removes chained pseudo classes", () => {
+      assert.equal(
+        h.fixClass(".visited:tw-bg-teal-100:nth-child(odd):active"),
```
Comment (sparksp) [inline test/tests.js:83]: Thanks for this regression test.

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "postcss-elm-tailwind",
   "main": "./index.js",
-  "version": "0.10.0",
+  "version": "0.10.1",
```
Comment (sparksp) [inline package.json:4]:
```suggestion
  "version": "0.10.0",
```

Changes to the version are handled by our release process already.

File: helpers.js (hunk 1 above)
Comment (icidasset) [reply helpers.js]: Yeah, you're totally right. Removed these changes 👍

**Pull request merged**